packages/components/src/Badge/Badge.snap.test.js

```diff
@@ -3,12 +3,11 @@ import renderer from 'react-test-renderer';
 
 import Badge from './Badge.component';
 
-describe('Badge', () => {
-	it('should render Badge without icon in outline style', () => {
+describe('Engine', () => {
```

Comment:
Why renaming it as Engine.

Reply:
because
Badge should render Badge does not make sense to me :)

Reply:
Yes but "Engine" makes even less sense.
You could keep "Badge" and rename the `it` test to "should render".